src/apis/cache.rs

```diff
@@ -1,7 +1,18 @@
-// For now all
+// TODO: Ideally we should remove files carefully, there may be some configuration files as well
+// We should only remove files which are present in sitemap + FPM.ftd + .packages
+
+// Note: If any read request in the system, It should not delete files.
+// In this case need to show the error, can't remove the files.
+
 pub async fn clear(req: actix_web::HttpRequest) -> actix_web::Result<actix_web::HttpResponse> {
-    // Ideally we should remove files carefully, there may be some configuration files as well
-    // We should only remove files which are present in sitemap + FPM.ftd + .packages
+    // Handle options
+    // 1. files=<file-path>
+    // 2. packages
+    //  - main
+    //  - all: remove main + .packages
+    //  - packages: will remove all packages
+    //  - name of the packages like: fpm.dev,foo,c, only name of the packages, entries from .packages folder
+    //  - can remove individual packages files, <package-name>/<file path>
```

Comment:
I recommend, `file=a.ftd&package=main&package=<package-name>&all-dependencies=false`

Note that individual file can also belong to any package, so `a.ftd` is wrong, we should pass `<package-name>/a.ftd`.

Reply:
Please check: https://github.com/FifthTry/fpm/discussions/475